python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -314,6 +320,50 @@ def __applyColumnVisibility( self ) :
 		finally :
 			self.__ignoreColumnResized = False
 
+	def __canReorderRows( self ) :
+
+		rowsPlug = self._qtWidget().model().rowsPlug()
+		if Gaffer.MetadataAlgo.readOnly( rowsPlug ) :
+			return False
+
+		if isinstance( rowsPlug.node(), Gaffer.Reference ) :
+			reference = rowsPlug.node()
+			for row in rowsPlug.children()[1:] :
```

Comment:
I guess there's already plenty of places where we document that rowsPlug[0] is the special default row, but this still confused me for a second until I remembered that.

Reply:
Clarified in c32338a